docs/build/micro-rdk/movement-sensor/gyro-mpu6050.md

```diff
@@ -6,7 +6,7 @@ type: "docs"
 description: "Configure an MPU-6050 movement sensor."
 images: ["/icons/components/imu.svg"]
 aliases:
-  - /micro-rdk/movement-sensor/gyro-mpu6050/
+  - "/components/movement-sensor/mpu6050/"
```

Comment:
Naive question: I don't understand what this line means or why it changed. What's going on here?

Reply:
This alias entry simply redirects any incoming requests for`https://docs.viam.com/build/micro-rdk/movement-sensor/gyro-mpu6050/` (currently live, will be gone after this PR) to `https://docs.viam.com/build/components/movement-sensor/gyro-mpu6050/` (not yet live, will be live after this PR) instead.

Reply:
It's not obvious to me that this PR is supposed to have a change like that. Why is it in here? I thought we were removing mention of "i2cs" and "spis" config entries in the boards, not renaming the gyroscope.

but if this is just some little nit that was supposed to go in wherever's convenient, I'm fine with it going here, too. :shrug:

Reply:
@sguequierre looks like you're intending to delete one of `/build/micro-rdk/movement-sensor/gyro-mpu6050.md` or `/components/movement-sensor/imu/mpu6050.md` ? Maybe I misinterpreted your intent with my comment to Alan above?

It looks like one of these two needs to be deleted. Are we moving this movement sensor out of the `micro-rdk` section into `components` like the rest?

Reply:
This sensor should be in both the micro- and regular RDKs. and since the codebases are separate, it's possible they have slightly different implementations and might need slightly different docs. Is this about unifying them to a single set of docs? I don't know what's happening in micro-rdk...

Reply:
Gotcha, thanks for the clarification. I'll let Sierra chime in with her intent then. Likely my misinterpretation, then, current PR has both file retained 👍

Reply:
The file at `/components/movement-sensor/mpu6050/` still exists. If it should stay - which is what penguinland is saying, we don't that second redirect. We do need the first one you deleted/edited.

Reply:
Sorry guys, I think this was a rebase issue that slipped past me! I didn't mean to change an aliases. I edited this file at all in error, reverting now...

Reply:
Thanks! Sorry for my confusion!